tests/pfcwd/test_pfcwd_warm_reboot.py

```diff
@@ -456,6 +456,9 @@ def pfcwd_wb_test_cleanup(self, setup_pfc_test):
         if not has_neighbor_device(setup_pfc_test):
             return
 
+        if not hasattr(self, "storm_threads"):
```

Comment:
What's the purpose of this path? Are there any known issues with it?

Reply:
Yes, self.storm_threads is set in pfcwd_wb_helper(). 
If case is skipped in test_pfcwd_wb(), pfcwd_wb_helper() won't be executed, and self.storm_threads will be None.

Reply:
Understood, thanks